### ISSUE TITLE
Should not trigger on route decorator with path parameter

### ISSUE THREAD

**Issue by Afizullah** [MichaelKim0407/flake8-use-fstring#23]
```python
from fastapi import FastAPI

app = FastAPI()

@app.get('/items/{item_id}') # FS003
async def read_item(item_id: int):
    return {'item_id': item_id}
```
I get app/main.py:5:10: FS003 f-string missing prefix which should not happen :/

**Comment by MichaelKim0407** [comment]
Please go through README and make sure you understand the designed behavior of this plugin.
